pages/common/gnmic-get.md

```diff
@@ -0,0 +1,20 @@
+# gnmic get
+
+> Get a snapshot of a gnmi network device operation data.
+> More information: <https://gnmic.kmrd.dev/cmd/get>.
+
+- Get a snapshot of the device state at {{path}}:
+
+`gnmic -a {{ip:port}} get --path {{path}}`
+
+- Query the device state at multiple paths:
+
+`gnmic -a {{ip:port}} get --path {{path1}} --path {{path2}}`
```

Comment:
```suggestion
`gnmic --aaddress {{ip:port}} get --path {{path1}} --path {{path2}}`
```

For paths we use `{{path/to/directory}}`, `{{path/to/file}}` generic placeholders. I don't know semantics of this path so please decide how better to fix it yourself. ;)

Also it's better to provide a concrete value instead of `{{ip:port}}` here because `--address` describes the meaning of the following argument.

Reply:
I'm not sure what to write for the paths, (example paths could be `"/"`, `/`, `info:/`, `info:/network_instances`, `/state/port[port-id=1/1/c1/1]/statistics`, `info:/port[port-id=1/1/c1/1]/statistics`) so I think that just leaving `{{path}}` is ok. 

I'm not sure what is meant for concrete value?

Reply:
Just leave it as is if you don't know what concrete value to provide. Basically, I don't know too. 🤔